.codecov.yml

```diff
@@ -11,3 +11,7 @@ coverage:
         if_ci_failed: success
         target: auto
         threshold: 0.01%
+    patch: 
+      default:
+        if_ci_failed: success
```

Comment:
A  comment of why we do this here would be not bad. So in the future maintainer understand why this was done.

Reply:
I commented everything